RangeRequest.bs

```diff
@@ -10,26 +10,218 @@ Editor: Chris Lilley, W3C, https://svgees.us/, w3cid 1438
 Editor: Myles C. Maxfield, Apple Inc., mmaxfield@apple.com, w3cid 77180
 Editor: Garret Rieger, Google Inc., grieger@google.com
 Abstract: Specification for incremental font transfer via HTTP range requests.
-Status Text: This is a largely empty document because we have just started working on it.
+Status Text: This is an editor's draft.
 </pre>
 
+<h2 id="intro">
+Introduction</h2>
 
-Introduction {#intro}
-=====================
+<em>This section is not normative.</em>
+
+Incremental Font Transfer (IFT) is a collection of technologies to improve the latency of remote fonts (or "web fonts") on the web. Without this technology, a browser needs to download every last byte of a font before it can render any characters using that font. IFT allows the browser to download only some of the bytes in the file, thereby decreasing the perceived latency between the time when a browser realizes it needs a font and when the necessary text can be rendered with that font.
```

Comment:
Meta: see [Path Forwards: one, two, three specs?](https://github.com/w3c/IFT/issues/48)

Reply:
Can we do this as a follow-up? If I move it now before the PR is landed, all the conversations on the PR would be marked as obsolete.